### ISSUE TITLE
Importing micro-memoize from a Typescript NodeJS environment

### ISSUE THREAD

**Issue by Emsu** [planttheidea/micro-memoize#99]
Getting this error with the recommended import from `tsc --build`
`import memoize from "micro-memoize/mjs";`

```
error TS2307: Cannot find module 'micro-memoize/mjs' or its corresponding type declarations.

1 import memoize from "micro-memoize/mjs";
                      ~~~~~~~~~~~~~~~~~~~
```

It looks like the types are bundled with Node in the mjs folder. The types are packaged with the root library. Is there something I'm missing or a way for me to fix this import?

**Comment by planttheidea** [comment]
Ah yeah, the general typings setup is in the top-level `index.d.ts`, but when deep-linking it does not see that. A super easy solution would be to copy over the `index.d.ts` file to `mjs/index.d.ts` and it should pick it up. Lemme start work on it, and I can bundle it in with the `4.1.0` release already planned with some internal typing changes.

**Comment by planttheidea** [comment]
Due to the bugfix nature of this, I decided to publish the fix prior to `4.1.0`. `4.0.15` has been released and should provide a fix for this. If you continue to have issues, lemme know!

**Comment by Emsu** [comment]
@planttheidea Thank you! I'll test it out tonight
